pkg/operator/ceph/csi/controller.go

```diff
@@ -270,6 +281,23 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 		}
 	}
 
+	if disableCSI {
+		err := r.stopDrivers()
+		if err != nil {
+			return opcontroller.ImmediateRetryResult, errors.Wrap(err, "failed to stop csi Drivers")
+		}
+		RBDDriverName = fmt.Sprintf("%s.rbd.csi.ceph.com", r.opConfig.OperatorNamespace)
+		CephFSDriverName = fmt.Sprintf("%s.cephfs.csi.ceph.com", r.opConfig.OperatorNamespace)
+		NFSDriverName = fmt.Sprintf("%s.nfs.csi.ceph.com", r.opConfig.OperatorNamespace)
+		driverNames := []string{RBDDriverName, CephFSDriverName, NFSDriverName}
+		for _, driverName := range driverNames {
+			err = r.transferCSIDriverOwner(r.opManagerContext, driverName)
+			if err != nil {
+				return opcontroller.ImmediateRetryResult, errors.Wrapf(err, "failed to create update %s driver for csi-operator driver CR", driverName)
```

Comment:
```suggestion
				return opcontroller.ImmediateRetryResult, errors.Wrapf(err, "failed to create update %s driver for csi-operator driver CR", driverName)
```

The log message does not match with the actual operation being done.